ledger/src/entry.rs

```diff
@@ -204,7 +222,7 @@ pub trait EntrySlice {
     /// Verifies the hashes and counts of a slice of transactions are all consistent.
     fn verify_cpu(&self, start_hash: &Hash) -> EntryVerifyState;
     fn start_verify(&self, start_hash: &Hash) -> EntryVerifyState;
-    fn verify(&self, start_hash: &Hash) -> bool;
+    fn verify(&self, start_hash: &Hash) -> EntryVerificationStatus;
```

Comment:
Similar to my other comment, leave this as bool. It makes it seem like you could return Pending from these interfaces but that's not actually the case because this is meant to be a blocking call.